libs/journeys/ui/src/libs/algolia/SearchBarProvider/SearchBarProvider.tsx

```diff
@@ -17,6 +21,12 @@ export interface SearchBarState {
    * selected languages sorted by continent
    */
   continentLanguages: Record<Continent, Language[]>
+  allContinentLanguages: Record<Continent, Language[]>
```

Comment:
Have been reflecting on the naming - thoughts on doing `selectedContinentLanguages` and `defaultContinentLanguages` or something along the lines?

Reply:
Good shout. I like `selectedContinentLanguages`. If we don't like `allContinentLanguages`, what about `continentLanguageMap` or `continentLanguageDirectory` to indicate **selectedContinentLanguages** is only for which are selected, and the **continentLanguageMap** is the full suite of languages mapped to the correct continent to lookup?

Reply:
Ah, I think I've got something that should be clear. Note I have swapped the order of the values here compared to .
```suggestion
  /**
   * only the selected languages sorted by continent
   */
  selectedContinentLanguages: Record<Continent, Language[]>
  /**
   * all the languages sorted by continent
   */
  continentLanguages: Record<Continent, Language[]>
```